src/Symfony/Component/Mailer/Bridge/Sendinblue/Transport/SendinblueSmtpsTransport.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mailer\Bridge\Sendinblue\Transport;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Mailer\Transport\Smtp\EsmtpTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+
+/**
+ * @author Yann LUCAS
+ */
+final class SendinblueSmtpsTransport extends EsmtpTransport
```

Comment:
This class should not be needed. The EsmtpTransport is able to determine the port automatically.

Reply:
I have removed this class.